.github/workflows/test-endroid.yml

```diff
@@ -10,27 +10,11 @@ jobs:
 
     strategy:
       matrix:
-        php-version: ['8.0', '8.1']
+        php-version: ['8.1', '8.2']
         endroid-version: ["^4"]
```

Comment:
`endroid-version@^3` is still supported by php 8.0+, and should probably be tested if BC is important. Otherwise, should remove the `$this->endroid4` checks from the `EncroidQrCodeProvider`...

Reply:
I have addressed your remarks, except this one, which can be dealt with later.